src/App.jsx

```diff
@@ -1,12 +1,46 @@
 import './App.css';
 
+import worldMap from './assets/world_map.png';
+import Countries from './pages/Countries/Countries.jsx';
+import { useState } from 'react';
+import Search from './pages/Search/Search.jsx';
+
 function App() {
+  let [applicationMode, setApplicationMode] = useState('regions');
+
+  function switchApplicationModes() {
+    (applicationMode === 'regions')
+    ? setApplicationMode('search')
+    : setApplicationMode('regions');
+  }
+
+  return (
+    <>
+        {
+          (applicationMode === 'regions')
```

Comment:
Dynamisch renderen, love it. Met Routing wordt het nog gemakkelijker binnenkort!